ReleaseNotes.md

```diff
@@ -18,6 +18,72 @@ Release Notes
 OpenRocket 22.02 (2023-02-XX)
 ------------------------
 
+The 22.02 release includes hundreds of new features, bug fixes, and UI improvements, more than we could ever fit into one set of release notes.  These notes summarize the highlights of the entire release; for more detail consult the notes from the five previous public beta releases.
+ 
+Please note that version 22.02 is required for Macs running MacOS 13.0 or later.
+ 
+## New Features
+* Rocket design features
+  * Pods and strap-on boosters
+  * Tail cones for pods
+  * Freeform fins on nose cones and transitions
+  * Highly configurable rail buttons
+  * Massive update to parachute configuration and part library
+  * Dave Cook's extensive component library now built-in
+  * Coefficient of Drag Override for components and assemblies (and more flexible override settings in general)
+  * Lots more detailed geometry warnings which identify the components involved
+* Appearance updates
+  * Settable opacity per-component
+  * Separate inner/outer appearance for tubes
+  * Separate left/right appearance for fins
+* App Infrastructure
+  * Packaged installers for Windows, Linux, and Mac (JAR file still available)
+  * Updated to Java11
+  * Native ARM version for Apple Silicon Macs (snappy!)
+  * File association for ORK files: double-click files to open in the app
+
```

Comment:
Maybe mention something about OR not quitting when closing the last design on macOS? Not super important and not strongly opinionated on this, so you decide whether to add it or not.